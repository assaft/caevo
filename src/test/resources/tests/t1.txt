John died after 1945.