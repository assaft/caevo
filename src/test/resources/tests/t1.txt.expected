ei1,died
t1,1945
l1,ei1,t1,AFTER